diff --git a/src/assistant/qcollectiongenerator/qcollectiongenerator.pro b/src/assistant/qcollectiongenerator/qcollectiongenerator.pro
index 23d32c6..67e66c5 100644
--- a/src/assistant/qcollectiongenerator/qcollectiongenerator.pro
+++ b/src/assistant/qcollectiongenerator/qcollectiongenerator.pro
@@ -16,3 +16,7 @@ SOURCES += ../shared/helpgenerator.cpp \
     ../shared/collectionconfiguration.cpp
 HEADERS += ../shared/helpgenerator.h \
     ../shared/collectionconfiguration.h
+
+# Link with QtHelp
+# Requires linking with QtCLucene
+LIBS += -L../../lib -L../../lib/fulltextsearch -lQtCLucene
