diff --git a/src/assistant/qhelpconverter/qhelpconverter.pro b/src/assistant/qhelpconverter/qhelpconverter.pro
index f875476..55f5b35 100644
--- a/src/assistant/qhelpconverter/qhelpconverter.pro
+++ b/src/assistant/qhelpconverter/qhelpconverter.pro
@@ -45,3 +45,7 @@ FORMS   += inputpage.ui \
            outputpage.ui
 
 RESOURCES += qhelpconverter.qrc
+
+# Link with QtHelp
+# Requires linking with QtCLucene
+LIBS += -L../../lib -L../../lib/fulltextsearch -lQtCLucene
